### PR TITLE
Set `-uroot-source` for the Mullvad specific targets as well

### PR DESCRIPTION
**Pull request by ln5**
The `-uroot-source` command line argument is needed in modern u-root, see u-root/README.

### CODE REVIEW

**Review by jbjorkang — APPROVED** [review]

**Comment by ln5** [comment]
I see that @jbjorkang committed c01c349 with this change to main.
If you have write access, why not merge this PR?
Trying to understand your preferred work flow.

**Comment by ln5** [comment]
Changes are already in main, since long.

**Pull request closed without merging**
